docs/conf.py

```diff
@@ -173,6 +173,10 @@
     # TODO: remove once we support different rtd config
     # files per project.
     "conf_py_path": f"/docs/{docset}/",
+    "display_github": True,
+    "github_user": "readthedocs",
+    "github_repo": "readthedocs.org",
+    "github_version": "main",
```

Comment:
I'm not 100% sure we always want to hard code this across all versions and PR builds, but seems fine for now.